doc/source/mast_data_products/pipeline-astrometric-calibration-description.rst

```diff
@@ -656,6 +656,70 @@ these computations were being performed in based up to the parent directory to
 replace the previously updated versions of the input files.  This entire sub-directory
 then gets deleted, unless the processing was being run in debug mode.
 
+Fit quality and selection of WCS solution
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The primary WCS selected will depends on several factors. These include the ``fit_rms``, 
+and whether a solution is deemed *compromised* (discussed below). The ``fit_quality`` is an integer value 
+between 1 and 5, where 1 is the best fit and 5 is the worst. As *a posteriori* WCS solutions 
+are fit with different catalogs (e.g. GAIADR3, GAIADR2, etc.), and different fit geometries 
+(e.g. rscale, shift, etc.), the code will only adopt the new solution if it has a 
+fit_quality less than or equal to the previous value AND a lower fit_rms. The fit_quality 
+value is determined as follows:
+
+  * 1 = Valid solution with fit_rms < 10 mas
+
+  * 2 = Valid but compromised solution with fit_rms < 10 mas
+
+  * 3 = Valid solution with fit_rms >= 10 mas
+
+  * 4 = Valid but compromised solution with fit_rms >= 10 mas
+
+  * 5 = No valid solution
+
+  * -1 = Alignment code failed altogether. 
+
+
+There are several ways in which a WCS solution can be considered compromised. 
+These manifest as a series of checks done in the align.py function 
+*determine_fit_quality()*. Checks must return True, otherwise the solution is 
+considered compromised. These checks include:
+  
+  * **consistency_check** (pipeline processing only): Ensures that the 
+    combination of the standard deviation of the X and Y shifts is less than one 
+    of two values, whichever is larger. The limit is either the value defined 
+    in align_pars["determine_fit_quality"]["MAX_FIT_RMS"] (default is 20 pixels) 
+    or the mean of the *fit_rms* values for all observations, whichever is larger. 
+    This check is disabled for SVM and MVM processing, as there can be large 
```

Comment:
Modify this comment as the "consistency_checks" are quoted as being for pipeline processing only.  Change to something like "The consistency check uses some lower-level functionality also employed by SVM and MVM processing.  For SVM and MVM process, this check is disabled..."

Reply:
Do you know where elements of this code are used for the SVMs and MVMs? If that's true, I should probably include where that code is located.